Assets/Scripts/Game Controllers/CardManager.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CardManager : MonoBehaviour
+{
+    [SerializeField]
+    private PlayerController player;
+
+    public bool PlayCard(string card)
+    {
+        switch (card)
+        {
+            case "LeftCard":
```

Comment:
Instead of using a card name and `CardManager` to decide what functionality to use when a card is played, `CardHandler` could be an abstract class with card subclasses that each have their own implementation of an abstract `TryPlayCard` method.
The Card Handler component on the Card prefab cannot be reused with that implementation, but I think doing that is simpler overall.

Reply:
So each card would have it's own script which implements the abstract card class? Or is there a way to have one script with all these classes on it?

Reply:
It could all be in one script by calling a method based on the value of a name variable or using an enum to distinguish card types. I think it's easier to maintain the code and add new cards when card behaviors are in separate scripts.